packages/flutter_tools/test/channel_test.dart

```diff
@@ -19,8 +19,10 @@ void main() {
       expect(await runner.run(<String>['channel']), 0);
       BufferLogger logger = context[Logger];
       expect(logger.errorText, hasLength(0));
-      expect(logger.statusText, contains('channels'));
-      expect(logger.statusText, contains('master'));
+      expect(logger.statusText, contains('Flutter channels:'));
+      // too flaky on bots
```

Comment:
It appears that sometimes the bots return an empty list (network hiccup?) and that when run locally you may not have 'master'.